esp/envoy-config.yaml

```diff
@@ -19,6 +19,8 @@ static_resources:
   - name: restlistener
     address:
       socket_address: { address: 0.0.0.0, port_value: 8081 }
+    # Request buffer size limit
+    per_connection_buffer_limit_bytes: 1073741824 # 1GB
```

Comment:
How do we make this work also in ESP v1? Or shall we try to move to ESP v2?

Reply:
I think we should, envoy is much better built than nginx as a proxy. 

For espv2, this is the docker lag:  --envoy_connection_buffer_limit_bytes
https://github.com/GoogleCloudPlatform/esp-v2/pull/344/files